### DISCUSSION TITLE
What is the correct specimenSymmetry for Aluminum with FCC structure？

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#1662] [Ask Anything]
Hi，
    I am wondering what the right specimenSymmetry parameter for Al?
    1 or 222?

     Tnanks!

**Comment by smallrain1996** [comment]
specimenSymmetry('orthorhombic')

**Reply by Brian-Liu-rx** [reply]
Thanks a lot

**Comment by ralfHielscher** [comment]
The specimen symmetry does not depend on your mineral (e.g. Al) but on the process your specimen has been formed. If you analyze an Al sheet directly after rolling it might have orthotropic specimen symmetry. In general it is suggested to use triclinic specimen symmetry.

Ralf.

**Reply by Brian-Liu-rx** [reply]
Thanks for reply！ What is the number for triclinic specimen symmetry? By using specimenSymmetry('triclinic ')? 
I usually use specimenSymmetry('1') or specimenSymmetry('222')

**Reply by ralfHielscher** [reply]
specimenSymmetry('1') and specimenSymmetry('triclinic ') are exactly the same.
Ralf.

**Reply by Brian-Liu-rx** [reply]
Thank you Ralf!  You are very nice!
